enhancements/storage/storage-hypershift.md

```diff
@@ -0,0 +1,973 @@
+---
+title: storage-hypershift
+authors:
+- "@jsafrane"
+reviewers:
+- "@gnufied"
+- "@dobsonj"
+- "@csrwng"
+approvers:
+- "@derekwaynecarr"
+api-approvers:
+ - None
+creation-date: 2022-07-26
+last-updated: 2022-07-27
+tracking-link:
+- https://issues.redhat.com/browse/STOR-962
+see-also:
+replaces:
+superseded-by:
+---
+
+# Running storage operators in HyperShift management cluster
+
+## Summary
+
+This enhancement describes how OCP storage components will run in HyperShift's managed control plane and in the guest
+cluster.
+
+The components currently are:
+
+* cluster-csi-snapshot-controller-operator and its operand, csi-snapshot-controller.
+* cluster-storage operator and its operand aws-ebs-csi-driver-operator and its operand AWS EBS CSI driver.
+  * All other CSI driver operators + CSI drivers should follow the same approach, AWS EBS is just the first one to
+    run its control plane in HyperShift's management cluster.
+  
+## Motivation
+
+A guest cluster in HyperShift should ideally run only user workloads, and it should not be possible to break the cluster
```

Comment:
Can we add link to https://hypershift-docs.netlify.app/reference/goals-and-design-invariants/

Reply:
Link added.